lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -9219,6 +9219,11 @@ void EmitForInOrForOf(ParseNode *loopNode, ByteCodeGenerator *byteCodeGenerator,
         return;
     }
 
+    Js::ByteCodeLabel skipThrow = byteCodeGenerator->Writer()->DefineLabel();
+    byteCodeGenerator->Writer()->BrReg2(Js::OpCode::BrNeq_A, skipThrow, loopNode->sxForInOrForOf.pnodeObj->location, funcInfo->undefinedConstantRegister);
```

Comment:
`BrNeq_A` is not strict check. So it will check both undefined and null. Is that what you asked?